src/main/java/net/snowflake/client/core/SecureStorageWindowsManager.java

```diff
@@ -104,12 +104,12 @@ public String getCredential(String host, String user, String type) {
 
       if (SecureStorageWindowsCredentialType.typeOf(cred.Type)
           != SecureStorageWindowsCredentialType.CRED_TYPE_GENERIC) {
-        logger.info("Wrong type of credential. Expected: CRED_TYPE_GENERIC", false);
+        logger.warn("Wrong type of credential. Expected: CRED_TYPE_GENERIC", false);
         return null;
       }
 
       if (cred.CredentialBlobSize == 0) {
-        logger.info("Returned credential is empty", false);
+        logger.debug("Returned credential is empty", false);
```

Comment:
warn?

Reply:
I am not sure, for me it looks like the case when we initially look into cache and there is no entry